src/System.Memory/src/System/Buffers/ReadOnlySequence_helpers.cs

```diff
@@ -166,24 +175,30 @@ private static SequencePosition SeekMultiSegment(object startSegment, int startI
 
                     if (memoryLength > count || (memoryLength == count && isCurrentAtEnd))
                     {
+                        // Fully contained in this segment; or at end of segment but isn't a next one to move to
```

Comment:
"... but **there** isn't a next one to move to"

Reply:
Done